driver/pgdriver/safe.go

```diff
@@ -1,11 +0,0 @@
-// +build appengine
```

Comment:
@vmihailenco @Aoang 

Do we need to implement any special logic for Google App Engine?

Reply:
The internal package has compatible.

https://github.com/uptrace/bun/blob/v1.2.6/internal/safe.go

https://github.com/uptrace/bun/blob/v1.2.6/internal/unsafe.go